packages/Caching/ValueObject/Storage/FileCacheStorage.php

```diff
@@ -5,15 +5,13 @@
 namespace Rector\Caching\ValueObject\Storage;
 
 use Nette\Utils\Random;
-use Nette\Utils\Strings;
-use Rector\Caching\ValueObject\CacheFilePaths;
```

Comment:
the `Rector\Caching\ValueObject\CacheFilePaths` class is un-used as of now and could be deleted